spec/features/staff/users_can_create_a_transaction_spec.rb

```diff
@@ -11,6 +11,26 @@
     before { authenticate!(user: user) }
     let(:user) { create(:beis_user) }
 
+    scenario "the form only shows relevant fields" do
+      activity = create(:programme_activity, :with_report, organisation: user.organisation)
+
+      visit activities_path
+
+      click_on(activity.title)
+
+      click_on(t("page_content.transactions.button.create"))
+
+      expect(page).to have_content t("page_title.transaction.new")
+      expect(page).to have_content t("form.label.transaction.value")
+      expect(page).to have_content t("form.legend.transaction.date")
+      expect(page).to have_content t("form.legend.transaction.receiving_organisation")
+      expect(page).to_not have_content t("form.label.transaction.description")
```

Comment:
I wrote this test to allow me to drive the changes on my code, and to make sure I wasn't leaving any non-relevant field in the form, but I see your point. I'll remove the `to_not have_content` expectations since it will definitely look odd in the future if I leave them there.